en_ID/pages/internship.md

```diff
@@ -130,6 +130,13 @@ To finish this step you can follow the tutorial below.
 
 ## Step 5 - Keeping fork updated
 
+At this fifth stage, we will discuss a little how to keeping fork updated. So the goal is how we can get the latest version from the upstream repository and then merge the new changes into your fork.
```

Comment:
This can be improved by adding some punctuation

At this fifth stage, we will discuss a little how-to to keeping fork updated. So the goal is how we can get the latest version from the upstream repository and then merge the new changes into your fork.

Note: you can follow these steps.

1. Add the original remote URL repository with this command git remote add upstream <REPO_URL>, the original repository is mostly called upstream.
2. To make sure remote is added, you can check with this command git remote -v.
3. Now, to keep upstream updated run this command git fetch upstream.